interfaces/builtin/pulseaudio.go

```diff
@@ -20,30 +20,164 @@
 package builtin
 
 import (
+	"bytes"
+
 	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/release"
 )
 
-const pulseaudioConnectedPlugAppArmor = `
+var pulseaudioConnectedPlugAppArmor = []byte(`
+/{run,dev}/shm/pulse-shm-* rwk,
```

Comment:
On the plugs side, shouldn't we use this instead: `owner /{run,dev}/shm/pulse-shm-* rwk,` ?